bin/heal

```diff
@@ -104,6 +104,9 @@ def main():
         VH.send_encodes()
         return None
 
+    # TODO: Data backup
```

Comment:
we're filtering videos based on `Course` model, but its not in imports. I wonder if this case has ever been executed.

Reply:
@Qubad786 It can be executed manually from the CLI, but in normal ETA operation it isn't